.github/workflows/main.yml

```diff
@@ -196,3 +200,51 @@ jobs:
             libzetasql-*.tar.gz
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  mac-release:
+    runs-on: macos-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    timeout-minutes: 120
+    env:
+      build_argv: '--features=-supports_dynamic_linker '
+      bazel_version: '3.7.2'
```

Comment:
reason ?

Reply:
> reason ?

zetasql need  bazel 3.7.2+  to build absl in macos